### PR TITLE
deb rpm: add helper script to repack debug packages

### PR DESCRIPTION
**Pull request by kenhys**
It is useful to investigate maintainer scripts.

### CODE REVIEW

File: fluent-package/test-repack-package.sh [hunk 1]
```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/bash
+
+#
+# Usage: test-repack-package.sh [OPTIONS] PATH_TO_DEB_OR_RPM
+#
+# e.g.
+# $ test-repack-package.sh debug fluent-package-6.0.0-1.el8.x86_64.rpm
+# $ test-repack-package.sh debug fluent-package_6.0.0-1_amd64.deb
+#
```
Comment (daipom) [inline fluent-package/test-repack-package.sh:9]: Thanks for this!
It looks convenient, but I don't understand what we can do by using this script...
Could you please reinforce these usage comments?

Comment (kenhys) [reply fluent-package/test-repack-package.sh]: Thanks, I've updated explanation

**Comment by daipom** [comment]
Thanks! I see!
This script adds debug messages to the existing package!

**Comment by daipom** [comment]
Should the script name be `test-repack-package`?
To me, it looks like a test script, not a helper script.
Wouldn't a name like `repack-package-with-debug-message`, for example, be easier to understand?
(This might be long, though...)

**Review by daipom — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**